buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -12,3 +12,8 @@ fun uuid(
     artifact: String = "uuid",
     version: String = "0.2.2"
 ): String = "com.benasher44:$artifact:$version"
+
+fun stately(
```

Comment:
Does kotlin have a dependency manager built in that knows to look for this file? Or is this a convention used by a third party dependency management system?

Reply:
Think I answered my own question. This is being used in gradle?

Reply:
> Think I answered my own question. This is being used in gradle?

Yup, `buildSrc` is a standard Gradle feature that allows us to add functionality to the Gradle build system. Here I'm just adding functions to produce the dependency Strings, and being in `buildSrc` allows them to be global.